src/utils/useCurrentPlatform.ts

```diff
@@ -0,0 +1,9 @@
+import { Platform } from '@/data/platforms';
+import { useRouter } from 'next/router';
+
+// Custom hook to return the current platform
+export function useCurrentPlatform() {
+  const router = useRouter();
+
+  return router.query.platform as Platform;
```

Comment:
Jacob confirmed this should work in Next.js's SSG mode

Reply:
yeah this is working with ssg, I'm seeing the link being generated correctly